docs/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Ipheco Chris Blog</title>
+    </head>
+    <body>
+        <section>
+            <h1>The Power of Meditation for a Healthier You</h1>
+            <img src="https://vegan.org/wp-content/uploads/2019/10/Meditation-Stock-600x492.jpg" alt="meditation image">
+            <p>
+                Meditation has been around for thousands of years and is
+                practiced by millions of people all over the world. While it is
+                often associated with spirituality and mindfulness, it also has
+                numerous health benefits. From reducing stress and anxiety to
+                improving sleep and boosting immunity, meditation is a powerful
+                tool that can help you live a healthier, happier life.
+            </p>
+
+            <p>
+                Do you want to read on more article on the above topics on
+                meditation, kindly click on any of these link below
+            </p>
+            <a name="top"></a>
```

Comment:
What does this anchor element do? It doesn't have href attribute  and content in between. If it's for easy navigation up and down the page, it should have an id attribute that is linked to another element  with target attribute value of the form #yourChosenId